MREUnityRuntime/MREUnityRuntimeLib/Assets/AssetCache.cs

```diff
@@ -41,7 +49,7 @@ public GameObject EmptyTemplate()
         }
 
         /// <inheritdoc cref="GetAssetIdsInSource"/>
-        public IEnumerable<Guid> GetAssetIdsInSource(AssetSource source)
+        public IEnumerable<Guid> GetAssetIdsInSource(AssetSource source = null)
```

Comment:
>null [](start = 74, length = 4)

I'm pretty certain TryGetValue will throw if source is null.

Reply:
You are correct, good catch!